trl/trainer/callbacks.py

```diff
@@ -35,13 +35,52 @@
 
 from ..models.utils import unwrap_model_for_generation
 from .judges import BasePairwiseJudge
-from .utils import decode_and_strip_padding, truncate_right
 
 
 if is_deepspeed_available():
     import deepspeed
 
 
+def _generate_completions(prompts: List[str], **kwargs: Dict[str, Any]) -> List[str]:
```

Comment:
I opted for `kwargs` here to give us some flexibility to add more things in future if needed and to avoid having a huge function signature. Happy to use explicit args if that's preferred

Reply:
ok, i'll revert

Reply:
done in https://github.com/huggingface/trl/pull/2061/commits/ea8de95d94531caf642438289f851dbee5b7824c